File: Codewars/8_kyu/Surface Area and Volume of a Box.js
// DESCRIPTION:
// Write a function that returns the total surface area and volume of a box as an array: [area, volume]

// My Solution:
function getSize(w, h, d) {
    const area = 2 * w * h + 2 * h * d + 2 * w * d
    const volume = w * h * d
    return [area, volume]
}

File: Codewars/Go/8_kyu/Surface Area and Volume of a Box.go
// Description:
// Write a function that returns the total surface area and volume of a box as an array: [area, volume]

// My Solution:
package kata

func GetSize(w,h,d int) [2]int {
  area := 2*w*h + 2*h*d + 2*w*d
  volume := w * h * d
  return [2]int{area, volume}
}